_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -437,8 +437,8 @@ public function update_data( $request ) {
 		// If $params was set via `get_body_params()` there may be some additional variables in the request that can
 		// cause validation to fail. This method verifies that each param was in fact updated and will throw a `some_updated`
 		// error if unused variables are included in the request.
-		$params = array_filter( $params, 'is_string' );
-		unset( $params['context'], $params['slug'] );
+		$filtered = array_filter( array_keys( $params ), array( $this, 'filter_unused_request_params' ) );
+		$params = array_intersect_key( $params, array_flip( $filtered ) );
```

Comment:
Wasn't it like that before this change? I'm not sure how `unset` works related to array keys.